diff --git a/src/tapi/include/tapi/Core/LLVM.h b/src/tapi/include/tapi/Core/LLVM.h
index c0d0d39..f223f95 100644
--- a/src/tapi/include/tapi/Core/LLVM.h
+++ b/src/tapi/include/tapi/Core/LLVM.h
@@ -15,6 +15,8 @@
 #ifndef TAPI_CORE_LLVM_H
 #define TAPI_CORE_LLVM_H
 
+#include <cstdint>
+
 #include "llvm/Support/Casting.h"
 
 namespace llvm {
diff --git a/src/tapi/include/tapi/LinkerInterfaceFile.h b/src/tapi/include/tapi/LinkerInterfaceFile.h
index aed033d..8ef3cca 100644
--- a/src/tapi/include/tapi/LinkerInterfaceFile.h
+++ b/src/tapi/include/tapi/LinkerInterfaceFile.h
@@ -14,7 +14,7 @@
 //===----------------------------------------------------------------------===//
 #ifndef TAPI_LINKER_INTERFACE_FILE_H
 #define TAPI_LINKER_INTERFACE_FILE_H
-
+#include <cstdint>
 #include <memory>
 #include <string>
 #include <tapi/Defines.h>
